stock_landed_costs_security/security/stock_landed_costs_security.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="0">
+    <record id="stock_landed_costs_security_category" model="ir.module.category">
+        <field name="name">Landing costs</field>
+        <field name="sequence">7</field>
```

Comment:
Still missing the `parent_id`

Reply:
I didn't see in your previous comments the parent_id.
which one should be?

Reply:
Please check this comment: https://github.com/OCA/stock-logistics-workflow/pull/1017#discussion_r908864135

Reply:
This one is still pending